operator/flags.go

```diff
@@ -315,5 +315,8 @@ func init() {
 	flags.String(option.BGPConfigPath, "/var/lib/cilium/bgp/config.yaml", "Path to file containing the BGP configuration")
 	option.BindEnv(option.BGPConfigPath)
 
+	flags.Bool(option.SkipCRDCreation, false, "when true, Kubernetes Custom Resource Definitions will not be created")
```

Comment:
This description should match the previous flag

Reply:
@kaworu hi! should I revert your suggestion in this case? https://github.com/cilium/cilium/pull/15655#discussion_r611664659

Reply:
Ah, I didn't see that Alex has already suggested this. Then I think keeping it is fine.

```suggestion
	flags.Bool(option.SkipCRDCreation, false, "When true, Kubernetes Custom Resource Definitions will not be created")
```

Reply:
done!